brave-http/src/main/java/com/github/kristofa/brave/http/HttpServerResponseAdapter.java

```diff
@@ -18,12 +18,8 @@ public HttpServerResponseAdapter(HttpResponse response)
 
     @Override
     public Collection<KeyValueAnnotation> responseAnnotations() {
-        int httpStatus = response.getHttpStatusCode();
-
-        if ((httpStatus < 200) || (httpStatus > 299)) {
-            KeyValueAnnotation statusAnnotation = KeyValueAnnotation.create("http.responsecode", String.valueOf(httpStatus));
-            return Arrays.asList(statusAnnotation);
-        }
-        return Collections.EMPTY_LIST;
+        KeyValueAnnotation statusAnnotation = KeyValueAnnotation.create(
```

Comment:
ps this is a logic change. It saves size to not log 2xx range responses. https://github.com/openzipkin/zipkin-java/blob/master/zipkin/src/main/java/zipkin/TraceKeys.java#L82